deepwell/src/services/user/service.rs

```diff
@@ -65,14 +65,23 @@ impl UserService {
             return Err(Error::UserSlugEmpty);
         }
 
-        // Check if username contains the minimum amount of required bytes.
-        if name.len() < ctx.config().minimum_name_bytes {
+        // Check if username contains the minimum amount of required bytes and chars.
+        let config = ctx.config();
+        if name.len() < config.minimum_name_bytes {
             error!(
                 "User's name is not long enough ({} < {})",
```

Comment:
`({} < {} bytes)` to be consistent with other places?

Reply:
Good catches for all of these. Fixing.